cloud/azurefunction/src/main/java/com/nashtech/function/util/CarUtil.java

```diff
@@ -0,0 +1,65 @@
+package com.nashtech.function.util;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import org.json.JSONObject;
+
+public class CarUtil {
+
+    private static String apiKey;
+
+    /**
+     * Converts the given mileage value from miles to kilometers per mile.
+     *
+     * @param mileage the mileage value in miles
+     * @return the updated mileage value in kilometers per mile
+     */
+    public static Double updateMileage(final Double mileage) {
+        Double mileageInKmps=mileage*1.6093440006147;
+        return mileageInKmps;
+    }
+
+    /**
+     * Converts the given price value to rupees.
+     *
+     * @param price the price value
+     * @return the updated price value in rupees
+     */
+    public static Double updatePrice(final Double price) {
+        Double priceInRupees=0.0;
+        try {
+            String apiUrl = "https://api.freecurrencyapi.com/v1/latest?apikey="+apiKey+"&currencies=INR";
```

Comment:
Do not leave URLs open. Keep it in a variable hence, it gets configurable without code changes.